### DISCUSSION TITLE
Weekly Challenges

### DISCUSSION THREAD

**Discussion by tityq** [ppy/osu#29168] [Ideas]
i love daily challenges, but i wish there was something similar that provides a harder challenge. here's my idea: weekly challenges, and here's basically how it would work:

- weekly challenge around 7-8* in SR after mods are enabled
- refreshes every thursday

in a sense its similar to how geometry dash does weekly demons

**Comment by Stoppedpuma** [comment]
Weekly challenge beatmaps will get harder the further in the week it is! You can expect some harder maps next week nearing the end.
